components/camel-platform-http-vertx/src/main/docs/platform-http-vertx.adoc

```diff
@@ -32,11 +32,10 @@ and the platform http component should auto-detect this.
 |`CamelVertxPlatformHttpRemoteAddress` |`io.vertx.core.net.SocketAddress` |The remote address for the connection if present on the Vert.x Web `RoutingContext`.
 |=======================================================================
 
-Camel also populates *all* request.parameter and Camel also populates *all* request.parameter and request.headers. For
-example, given a client request with the URL,
+Camel also populates *all* `request.parameter` and Camel also populates *all* `request.headers`.
+For example, given a client request with the URL,
 `\http://myserver/myserver?orderid=123`, the exchange will contain a
-header named `orderid` with the value 123.
-request.headers. For example, given a client request with the URL, `\http://myserver/myserver?orderid=123`, the exchange will contain a header named `orderid` with the value 123.
+header named `orderid` with value `123`.
```

Comment:
Hey @jamesnetherton, would you mind reviewing this part, please? It was duplicated, so I edited it. 

Thanks!